source/_components/sensor.waze_travel_time.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: "Waze Travel Time"
+description: "Instructions on how to add Waze travel time to Home Assistant."
+date: 2018-01-23 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: waze.png
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.61
```

Comment:
`0.61` is already shipped some time ago. Please set it to `0.63`

Reply:
Done